src/headless/celery_app.py

```diff
@@ -19,11 +20,13 @@
 )
 
 # Initialize qgis_app
-from safe.test.utilities import get_qgis_app  # noqa
-QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+from safe.test.qgis_app import qgis_app
+# initialize qgis_app
+APP, IFACE = qgis_app()
 
 # Load QGIS Expression
 from safe.utilities.expressions import qgis_expressions  # noqa
+qgis_expressions()
```

Comment:
cc @ismailsunni I assume you want to use it somewhere? so it had to be initialized?

Reply:
No. Just importing it makes the trick, to  make qgis aware about these expressions

Reply:
Yes, we don't need to initialize it. Confirmed by @Gustry :D

Reply:
How does it works exactly? I saw the file, there is no invocation at all. Only that qgis_expression function declaration. Does it works by hooking a predefined name?

Reply:
Ok. I understand after tracing the import. It works using decorators in the sub module

Reply:
Yes, the decorator will register these functions in QGIS, no need to call them explicitely